### ISSUE TITLE
Training logs

### ISSUE THREAD

**Issue by mitming** [cvlab-columbia/zero123#47]
Hi @ruoshiliu,

I tried to reproduce the results with 32 V100 GPUs (batch size is 12 for each node, and the accumulate_grad_batches is 4 ), could you please help check these losses and reconstruction results?

In addition, I really hope that you can provide the training logs for future research.

<img width="1370" alt="image" src="https://user-images.githubusercontent.com/73068772/234809562-3d5325d3-572b-4ed6-af1f-93faf311746a.png">
<img width="1036" alt="image" src="https://user-images.githubusercontent.com/73068772/234809731-f0932767-099f-438e-a4a5-b08a7aadd696.png">

**Comment by ruoshiliu** [comment]
Just curious, is 12 the maximum batch size for V100? Since you used grad_acc = 4 so effective you trained for around 3000 epochs with batchsize=1536. The results do make sense to me. The loss_epoch is also decreasing pretty consistently. Don't worry too much about the iteration loss. The variance is expected for training diffusion models.

**Comment by mitming** [comment]
> Just curious, is 12 the maximum batch size for V100? Since you used grad_acc = 4 so effective you trained for around 3000 epochs with batchsize=1536. The results do make sense to me. The loss_epoch is also decreasing pretty consistently. Don't worry too much about the iteration loss. The high variance is expected for training diffusion models.

Yes, the maximum batch size for V100 is 12 on each GPU. Otherwise, there would be GPU OOM. F.Y.I., I use the torch 2.0, it should be GPU memory efficient more than other torch versions.

In addition, I also noticed that doubling the batch size does not cause significant memory usage, and the model only appears as GPU OOM after training for a period of time. I guess this is because the Diffusion model is too large, maybe you can help to verify this conclusion on A100?

Finally, could you please release the training logs? It would be very helpful!

**Comment by ruoshiliu** [comment]
Hi @mitming , thanks for the info on GPU OOM. I think VRAM should scale linearly with batch size but with an offset of the memory occupied by loading the model weights on GPU. It's indeed quite hard to train or finetune stable diffusion on V100 without further optimization. The training logs will not be released sorry. Happy to help with other trouble (if any).

**Comment by mitming** [comment]
> Hi @mitming , thanks for the info on GPU OOM. I think VRAM should scale linearly with batch size but with an offset of the memory occupied by loading the model weights on GPU. It's indeed quite hard to train or finetune stable diffusion on V100 without further optimization. The training logs will not be released sorry. Happy to help with other trouble (if any).

Hi @ruoshiliu,

Thank you for your honesty and contribution, if there are other questions, I will create a new issue.

**Comment by yuanzhi-zhu** [comment]
@ruoshiliu 
Hello Ruoshi, is it possible to provide us with the final epoch loss from your experiment?

**Comment by yifanjiang19** [comment]
Hi @liming-ai ,

It's great to see you have tried torch2.0. I'm wondering whether you have met any issues when migrating to torch2.0? Thanks!

best,

**Comment by VLadImirluren** [comment]
@liming-ai Can you give me a brief explaination about the five line(train/conditioning train/inputs train/reconstruction train/samples train/samples_cfg_scale_3.00), 
I know input, but other 3 I can't understand. Such as reconstruction is same as input, condition may be as condition view.
But what is samples and samples_cfg_scale_3.00.How do I assess how good the training is by these two columns?

i will be grateful if you can give me some advice....
